tools/docker/chainlink-variables.env

```diff
@@ -0,0 +1,3 @@
+# Add in your environment variables here
+# if you want to add more environment variables not listed 
+# in .env
```

Comment:
Should this be added to git ignore, or can you clarify why this is separate form `tools/docker/.env`? Do environment variables here need to be added to an environment in `tools/docker/docker-compose.yaml`?

Reply:
Good feedback, ill add in a README for this directory explaining why this is here. It exists because this file allows you to merge in environment variables not listed under https://github.com/smartcontractkit/chainlink/pull/1937/files#diff-0b300c2f6d56ab220f65f362b2a3d7bbR10 to be accessable by that container. This is needed since there's many more environment variables not listed under that key, that we may need to add when debugging / testing, but its noisy to add them all. See https://docs.docker.com/compose/compose-file/#env_file